scripts/compare-deposits/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/nspcc-dev/neo-go/scripts/compare-deposits
+
+go 1.20
```

Comment:
should be compiled by the compiler of this version?

Reply:
Ah, `slices.DeleteFunc`. Let it just be 1.21, this script is not worth spending more time on it.